gen/definitions/trustsec_egress_matrix_cell.yaml

```diff
@@ -0,0 +1,61 @@
+---
+name: TrustSec Egress Matrix Cell
+rest_endpoint: /ers/config/egressmatrixcell
+doc_category: TrustSec
+id_path: response.id
+skip_minimum_test: true
```

Comment:
Why does a minimum test not work?

Reply:
due to issue with PUT method:  Failed to configure object (PUT), got error: HTTP Request failed: StatusCode
        400, Message: Validation Error - Mandatory fields missing: [Id,], {
          "ERSResponse" : {
            "operation" : "PUT-update-egressmatrixcell",
            "messages" : [ {
              "title" : "Validation Error - Mandatory fields missing: [Id,]",
              "type" : "ERROR",
              "code" : "Application resource validation exception"
            } ],